### DISCUSSION TITLE
Unread badges

### DISCUSSION THREAD

**Discussion by yossizahn** [filips123/PWAsForFirefox#373] [Ideas]
Hi,
I have not seen any discussion on this in this repo
Will it be possible to display an unread badge on the app icon?
Perhaps by using a polyfill for the [badging api](https://developer.mozilla.org/en-US/docs/Web/API/Badging_API)?

**Comment by filips123** [comment]
Do any big websites use Badging API? I know many messaging/social websites use some kind of "icon badge" for unread notifications, but I don't know if they only change the whole favicon themselves or also use the actual badging API. This is useful to know so I can decide if it's worth implementing that API, especially since it's currently still a draft and not an officially accepted standard.

**Reply by yossizahn** [reply]
Hi, thanks for the reply!

My use case is Google Chat. It can be installed as a PWA. They do use the badging API as far as I can tell.

**Reply by filips123** [reply]
I may implement this in the future, but I cannot guarantee when. As it requires integration with the operating system, it's probably quite hard to implement, and I'm not even sure if it's possible to do on every operating system. So, I'll probably wait until more websites start using this or I find a good way to implement it.

**Reply by abePdIta** [reply]
I agree, it would be useful! I use Gmail, but I believe that other websites use Badging API too.
Chrom* handle this on Windows and macOS (and possibly support Linux too), so I think that should be possible to take a look at their implementation.
Thanks for bringing back PWA to Firefox!

**Reply by TheWhomst** [reply]
This feature would be amazing! You have the best PWA I could find and that feature would be the cherry on top 🙏
